### DISCUSSION TITLE
R Workflow Report

### DISCUSSION THREAD

**Discussion by harrelfe** [quarto-dev/quarto-cli#880] [Show and Tell]
My R Workflow report [here](https://fharrell.com/post/rflow) is getting mature.  It showcases a lot of Quarto's formatting capabilities and delves into making a report using more advanced features (interactive plotly graphics, use of marginal notes) work well in LaTeX/PDF in addition to html.  The report also shows how to dynamically generate multiple Quarto tabs and several other tricks.

**Comment by bryanhanson** [comment]
Great resource Frank, and much of the content implicitly makes the argument about Quarto's design to take the best of all the other alternatives, smooth off the rough edges and make the syntax consistent (at least, that's how I look at the design and intention of Quarto).
